packs/joomla.js

```diff
@@ -15,7 +15,7 @@ const UIStrings = {
   /** Additional description of a Lighthouse audit that tells the user how they can improve image loading by using webp in the context of the Joomla CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
   'uses-webp-images': 'Consider using a [plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=webp) or service that will automatically convert your uploaded images to the optimal formats.',
   /** Additional description of a Lighthouse audit that tells the user how they can improve performance by lazy loading images that are initially offscreen in the context of the Joomla CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
-  'offscreen-images': 'Install a [lazy-load Joomla plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=lazy%20loading) that provides the ability to defer any offscreen images, or switch to a template that provides that functionality. Starting with Joomla 4.0 the core is shipping an dedicated lazyloading plugin please enable the "Content - Lazy Loading Images" plugin. Also consider using [an AMP plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=amp).',
+  'offscreen-images': 'Install a [lazy-load Joomla plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=lazy%20loading) that provides the ability to defer any offscreen images, or switch to a template that provides that functionality. Starting with Joomla 4.0, the core is shipping a dedicated lazyloading plugin ("Content - Lazy Loading Images") which you should enable. Also consider using [an AMP plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=amp).',
```

Comment:
cc @jeckodevelopment we're trying to fix the wording to make this more clear



from my understanding Joomla 4 is in beta but the final isn't shipped yet, right?

also... where can i read about these dedicated plugins that are bundled and shipped with joomla core?  looks like email-cloak and recaptcha are similar?

https://github.com/joomla/joomla-cms/pull/28838 is where this plugin was added.

cc @zero-24  to confirm: is this plugin shipped by default with joomla 4? also, it is disabled by default?


if a user were to enable it.. can you show a screenshot of where they'd find this?  i'm personally seeing the word "extension" more than the word "plugin".. so i want to be sure this is actually helping people.

Reply:
Hi, thanks for picking this up. Right now there is that plugin in the core beta. But i'm not sure whether that will be the final solution for the core. We might should better say something like that joomla 4 will be shipped with lazyloading supported in the core.

Reply:
I have just added the PR here: https://github.com/joomla/joomla-cms/pull/30748 for the Joomla Core which is going to drop the dedicated plugin in favor of an more deeper implementation that does not longer require an plugin.

I have also added an suggestion down for a better wording. Feel free to ping me on any questions.